pg_stream_copy/encoder.py

```diff
@@ -101,6 +101,8 @@ def __exit__(self, exc_type, exc_val, exc_tb):
     DataType.CHARACTER_VARYING: protocol.build_character_varying,
     DataType.TEXT: protocol.build_text,
     DataType.DATE: protocol.build_date,
+    DataType.TIMESTAMP: protocol.build_timestamp,
+    DataType.TIMESTAMP_TZ: protocol.build_timestamp,
```

Comment:
If there are two separate types, even identical, like text and character varying, we should provide and test two separate methods...